src/createServer.js

```diff
@@ -1,3 +1,75 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable operator-linebreak */
+function createServer() {
+  const http = require('http');
+
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const RESULT_DATA = {};
+    const TEXT = normalizedURL.pathname.slice(1);
```

Comment:
```suggestion
    const resultData = {};
    const text = normalizedURL.pathname.slice(1);
```
It's better to use camelCase for calculated variables